source/index.ts

```diff
@@ -58,6 +59,34 @@ wiki.search = async (query: string, searchOptions?: searchOptions): Promise<wiki
     }
 }
 
+/**
+ * Returns the search results for a given query
+ *
+ * @remarks
+ * Limits results by default to 10
+ *
+ * @param query - The string to search for
+ * @param autocompletionOptions - The number of results and if suggestion needed {@link autocompletionOptions | autocompletionOptions }
+ * @returns an array of {@link wikiSearchResult | wikiAutocompletionResult }
+ */
+ wiki.autocomplete = async (query: string, autocompletionOptions?: searchOptions): Promise<wikiAutocompletionResult> => {
+  try {
+      const autocompletionParams: any = {
+          'list': 'search',
+          'limit': autocompletionOptions?.limit || 10,
+          'search': query,
+          'action': 'opensearch',
+          'redirect': 'return'
+      }
+
+      const [_, autocompletions] = await request(autocompletionParams, false);
```

Comment:
can we check if we can avoid assigning _?

Reply:
sure, the payload of the OpenSearch-api looks like `[query, [suggestion1, suggestion2, ...]]`.
we could do `const [, autocompletions]` or `const response` and  `return response[1]`. what do you prefer?